packages/nodes-base/tsconfig.json

```diff
@@ -11,6 +11,7 @@
 		"module": "commonjs",
 		"removeComments": true,
 		"strict": true,
+		"skipLibCheck": true,
```

Comment:
why did you add this option? it has some caveats and does not work well with monorepo projects

Reply:
There were some type-related errors coming from dependencies, and it tends to be a pretty standard thing to use.
https://github.com/tsconfig/bases/blob/main/bases/node14.json

I'm not sure what other options there are besides making a pr for the packages that have issues.

Reply:
Hi @Ken-Michalak I did work today on your PR without this option and did not got any errors, can you please check changes that I made in https://github.com/n8n-io/n8n/pull/3244